experiments/convert_traj.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+"""
+Convert trajectories from `imitation` format to openai/baselines GAIL format.
+"""
+
+from typing import List
+
+import numpy as np
+
+from imitation.util import rollout as ro_util
+
+
+def convert_trajs_to_sb(trajs: List[ro_util.Trajectory]) -> dict:
+  """Converts Trajectories into the dict format used by Stable Baselines GAIL.
+  """
```

Comment:
One-liner docstrings do not normally have a newline before closing `"""`

Reply:
Doesn't fit in 80 characters here